lib/timeval.c

```diff
@@ -84,6 +84,37 @@ struct curltime curlx_tvnow(void)
   return cnow;
 }
 
+#elif defined(__APPLE__)
```

Comment:
Wouldn't it be better to just check for mach_absolute_time() in the configure script?

Reply:
Probably.

Reply:
There are two ways to configure _curl_: configure and cmake.  Are pull requests expected to update both?

Reply:
Preferred at least! Just adding a check for a function is pretty easy in both though.

Reply:
Done!  Well, except that it does not work with cmake.